README.md

```diff
@@ -1,16 +1,54 @@
 # Release Integration Tests
 
 
-## 1. FBC Test (happy path):
-The purpose of this release test (FBC) is to verify the functionality and stability of feature in RHTAP.
-Test is a "happy-path" test and it's running on the internal RHTAP stage cluster.
+## 1- Fbc happy path e2e-test ##
+
+ This section provides instructions on running the FBC e2e test with the provided script
+
+   ## Test Steps and Expected Result
+
+   1. **Pre-Running Test:** The following setup actions need to be completed before running the test script.
+      
+      The test is running against RHTAP-Stage, so a user who wants to monitor the test running through the UI or the CLI. 
+      will need access to the cluster and the workspaces *release-dev-team* and *release-managed-team*. 
+      
+      Our test expect to run on both workspaces above, and it's expected (based on the flow agreed), namespaces dev and managed exist, policy *(fbc-policy)*, service account, secrets already deployed on the two dedicated workspaces 
+      *release-dev-team* and *release-managed-team*.
+
+      Since this test is running against the RHTAP it requires the user running be logged in in SSO service, else it will pop up a SSo login web page.
+      
+
+   2. **Running the Test:**
+      The test script sets up resources, waits for pipeline runs to complete, checks release status, and reports the result.
+
+
+   3. **Expected Result:**
+
+      a. `We expect that a Build PipelineRun in dev workspace started and succeeded.`
+
+      b. `A PipelineRun is executed in the managed workspace using the fbc-release pipelineRef and it succeeds.`
+
+      c. `Release CR is expected to be updated with Released after Release PipelineRun complete and scceeds.`
+   ## How to Run the Test
+
+   To run the test script, follow these steps:
+
+   1. **Open a Terminal:** Open a terminal window on your system.
+
+   2. **Navigate to the Script Directory:** Use the `cd` command to navigate to the directory containing the script.
+
+         ```sh
+      cd fbc
+   3. **Make the Script Executable:** If the script is not already marked as executable, use the following command to grant execute permission:
+      
+      ```sh
+      chmod +x fbc-test.sh
+   4. **Run the script:** Excute the script using the command:
+
+      ```sh
+      ./fbc-test.sh
+   5. **Monitor the Test Output:** The script will provide output indicating the progress of the test, including setup actions, waiting for pipeline runs, and checking the release status, also can monitor the RHTAP-Stage Console.
```

Comment:
`s/and checking the release status, also can monitor the RHTAP-Stage Console./and checking the release status. One can also monitor the RHTAP-Stage Console./`